src/main/java/picard/sam/FastqToSam.java

```diff
@@ -328,11 +329,14 @@ protected int doWork() {
         final SAMFileWriter writer = new SAMFileWriterFactory().makeWriter(header, false, OUTPUT, REFERENCE_SEQUENCE);
 
         final FastqReader reader1 = fileToFastqReader(FASTQ.toPath());
+        final boolean pipedInput = !Files.isRegularFile(FASTQ.toPath());
         if (reader1.hasNext()) {
-            // Set the quality format
-            QUALITY_FORMAT = FastqToSam.determineQualityFormat(reader1,
-                    (FASTQ2 == null) ? null : fileToFastqReader(FASTQ2.toPath()),
-                    QUALITY_FORMAT);
+            if (!pipedInput) {
+                // Set the quality format
+                QUALITY_FORMAT = FastqToSam.determineQualityFormat(reader1,
+                        (FASTQ2 == null) ? null : fileToFastqReader(FASTQ2.toPath()),
```

Comment:
Is there a use case where we care about reading FASTQ *and* FASTQ2 from different named pipes?  Should we handle that?  Are single ended reads good enough for what you're doing?

Reply:
For our specific case, handling a single piped input is sufficient in that what we have is interleaved R1 and R2 reads from another tool. More generally, my feeling is that upstream processing of multiple inputs into a single stream with e.g. `paste -d '\n' f1 f2 ...` is more robust than trying to handle all the input type possibilities within this tool, at least without some substantial refactoring e.g. to handle `USE_SEQUENTIAL_FASTQS`.  I added a hint to the help text to suggest as much.

Reply:
I'm convinced.